src/sentry/notifications/notifications/integration_nudge.py

```diff
@@ -109,8 +109,5 @@ def build_attachment_title(self, recipient: RpcActor) -> str:
     def build_notification_footer(self, recipient: RpcActor, provider: ExternalProviders) -> str:
         return ""
 
-    def record_notification_sent(self, recipient: RpcActor, provider: ExternalProviders) -> None:
-        pass
-
```

Comment:
so the idea is that with this out it'll call one defined in the parent? 
https://github.com/getsentry/sentry/blob/7e3d3d02a1bde28d5ceb89d2da01d2c379017fb4/src/sentry/rules/actions/integrations/base.py#L90

Reply:
@vartec exactly, no idea why this was a pass. I blame a certain former ex-employee.